cypress/integration/main_page_spec.js

```diff
@@ -281,5 +281,48 @@ describe('Dashboard Home Page', () => {
 
       shouldHaveDeletedToken();
     });
+
+    it('adds the default datepicker options to the URL', () => {
+      const now = Date.UTC(2009, 0, 5, 13, 35, 3);
+      cy.clock(now);
+
+      cy.visit('/?new_session=true');
+      cy.url().should('include', 'startDateTime=20090104133500');
+    });
+
+    it('should not add the initial Datepicker state to the URL once cleared', () => {
+      const now = Date.UTC(2009, 0, 5, 13, 35, 3);
+      cy.clock(now);
+
+      cy.visit('/?new_session=true');
+      cy.url().should('include', 'startDateTime=20090104133500');
+
+      cy.get('[data-cy=datetime-clear]').click();
+      cy.contains('nav li a', 'Collections').as('collections');
+      cy.get('@collections').click();
+      cy.get('.logo').click();
+
+      cy.url().should('not.include', 'startDateTime=');
+    });
+
+    it('should update the Datepicker with the params in the URL', () => {
+      cy.visit('/?startDateTime=20081229133500&endDateTime=20090105133500');
```

Comment:
```suggestion
      cy.visit('/granules/?startDateTime=20081229133500&endDateTime=20090105133500');
```
This way we're testing an additional page beyond the homepage.  Take it or leave it, but I like it.

Reply:
Sure, that works.